ser/train.py

```diff
@@ -0,0 +1,42 @@
+import torch
+import torch.nn.functional as F
+
+
+def model_train(epochs, training_dataloader, validation_dataloader, 
```

Comment:
Nitpick - the formatting here is a bit ... non standard. There's a format command in vscode, quite nice to run taht over your code!

Reply:
Also, your training algorithm is very neat now, love it 👍